packages/hoprd/hopr-admin/client.js

```diff
@@ -0,0 +1,56 @@
+const API_ENDPOINT_PORT = 13301
+const API_ENDPOINT = `http://localhost:${API_ENDPOINT_PORT}/api/v2/`
+const API_SECURITY_TOKEN = "^^LOCAL-testing-123^^"
```

Comment:
we will need to create a different issue that expands the UI to allow for inputting the API URL, API PORT, and API token - for now these should be done via query parameters

Reply:
I have refactored AbstractCommand class to use a newly create fetching utility through `HoprFetcher` class which extends `HoprClient` class; as a result, `port` and `apiToken` can be passed as query params form the url.

For example:
```js
http://localhost:3000/?port=13301&apiToken=^^LOCAL-testing-123^^
```

And.. I have used the default of port=3301 and apiToken= ^^LOCAL-testing-123^^ as default values such that it will work out of the box as well:
```js
http://localhost:3000/
```